src/main/resources/com/mitre/iCalendar/xsd/iCalendar.dfdl.xsd

```diff
@@ -57,33 +58,35 @@
     -->
 	<xs:element name="icalendar">
 		<xs:complexType>
-			<xs:sequence dfdl:separator="%NL;" dfdl:separatorPosition="infix" dfdl:separatorSuppressionPolicy="trailingEmptyStrict">
-				<!--  
-						An iCalendar stream can contain one or more iCalendar objects.  Each
-   						iCalendar object, delimited by "BEGIN:VCALENDAR" and "END:VCALENDAR",
-   						is enclosed by the vcalendar XML element.
-					-->
-				<xs:element name="vcalendar" dfdl:initiator="BEGIN:VCALENDAR%NL;" dfdl:terminator="END:VCALENDAR" 
-					        maxOccurs="unbounded" dfdl:occursCountKind="parsed">
-					<xs:complexType>
-						<xs:sequence>
-							<xs:element ref="properties" />
-							<xs:element ref="components" />
-						</xs:sequence>
-					</xs:complexType>
-				</xs:element>
-				<xs:element name="endNL" type="xs:string" 
-					dfdl:lengthKind="explicit" dfdl:length="0"
-					minOccurs="0" dfdl:occursCountKind="implicit" />
+        	<xs:sequence dfdlx:layerTransform="lineFolded_iCalendar" dfdlx:layerLengthKind="implicit" dfdlx:layerLengthUnits="bytes" dfdlx:layerEncoding="us-ascii">
```

Comment:
Line is long. One property per line would be better indenting.

Reply:
Fixed. Also converted all tabs to 2 spaces.

Reply:
I'd actually prefer you not convert all tabs to spaces. It makes it difficult to determine what the actualy changes in a commit are.

Reply:
Also updated README.

Reply:
Note that if we did care about tabs vs spaces, it should be done as a separate commit so we dan differentiante the layer chnages vs the whitespace changes.

Reply:
The last commit has 1) update to the README and 2) formatting changes to the schema file